src/openrct2-ui/UiContext.Win32.cpp

```diff
@@ -9,22 +9,23 @@
 
 #ifdef _WIN32
 
-// Windows.h needs to be included first
-// clang-format off
-#    ifndef WIN32_LEAN_AND_MEAN
-#        define WIN32_LEAN_AND_MEAN
-#    endif
-#    include <openrct2/Diagnostic.h>
-#    include <windows.h>
-#    include <shellapi.h>
-    // clang-format on
+    #ifndef WIN32_LEAN_AND_MEAN
+        #define WIN32_LEAN_AND_MEAN
+    #endif
+
+    // clang-format off
```

Comment:
`clang-format` keeps changing its mind about this line's indentation. I forget what we did about this last time this happened. Any ideas?